src/featured-sponsors.json

```diff
@@ -96,6 +105,15 @@
     "url": "https://mobilistics.de/",
     "github": "mobilistics"
   },
+  {
+    "name": "optasy",
+    "type": "standard",
+    "logo": "/logos/optasy-full.svg",
+    "darklogo": "/logos/optasy-darkmode.svg",
+    "squareLogo": "/logos/optasy-square.svg",
```

Comment:
the preview shows the wrong file with this name. It's odd. Since the right one was on a previous PR.